### ISSUE TITLE
Projects feature - nit list

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#286]
Here are some items to be closed out for the projects feature before we can deploy it:

**1. Layout glitch in iOS**
![image](https://cloud.githubusercontent.com/assets/14217285/14421773/b091ab8e-ffd3-11e5-8b17-1326d0679c11.png)

**2. Copy task should copy old title and append "(copy)" / "(Kopie)" in text.**

**3. inside project, 'create task' button should** 
3.1 display project name as read-only
3.2 display work group name as read only
3.3 offer 'Cancel' / 'Abbrechen' as an option, styled as a link text
![image](https://cloud.githubusercontent.com/assets/14217285/14421892/3c37fcb0-ffd4-11e5-9867-a8007cf314a9.png)

**4. Button 'Copy task' is shown to user that is not WG organizer (of the project).** 
You can only copy a task if you have the right to edit it (i.e. it is in y WG managed by the user).
![image](https://cloud.githubusercontent.com/assets/14217285/14423653/2f067586-ffdd-11e5-8dae-78f76e55615d.png)

**5. Projects listing from leftnav does not adjust size in iOS**
![image](https://cloud.githubusercontent.com/assets/14217285/14424462/76d42d96-ffe1-11e5-8d5e-664359050686.png)

**Comment by phillipoertel** [comment]
I was able to fix some things and deployed on staging. Status:

1. Fixed
2. Should have already worked, and worked for me locally on staging (for de and en). It should even append copy 2, 3, ... to copies of copies.
3.1 Todo 
3.2 Todo 
3.3 Fixed
4. Todo (actually implemented but needs debugging)
5. Fixed

Unfortunately I won't have time to look into the open points before Friday. All my code is on master though and should be straight-forward to understand.
